src/containers/Experiment/index.js

```diff
@@ -1,17 +1,172 @@
+let Experiment = ({
+  fetch,
+  experiment,
+  addExperimentToDataset,
+  addSamplesToDataset
+}) => (
+  <Loader fetch={fetch}>
+    {({ isLoading }) =>
+      isLoading ? (
+        <div className="loader" />
+      ) : (
+        <div>
+          <Button text="Back to Results" buttonStyle="secondary" />
+
+          <div className="experiment">
+            <div className="experiment__accession">
+              <img
+                src={AccessionIcon}
+                className="experiment__stats-icon"
+                alt="Accession Icon"
+              />
+              {experiment.accession_code}
+            </div>
+
+            <div className="experiment__header">
+              <h3>{experiment.title || 'No Title.'}</h3>
+              <Button text="Add to Dataset" onClick={addExperimentToDataset} />
+            </div>
+
+            <div className="experiment__stats">
+              <div>
+                <img
+                  src={OrganismIcon}
+                  className="experiment__stats-icon"
+                  alt="Organism Icon"
+                />{' '}
+                {experiment.species}
+              </div>
+              <div>
+                <img
+                  src={SampleIcon}
+                  className="experiment__stats-icon"
+                  alt="Sample Icon"
+                />{' '}
+                {experiment.samples.length} Samples
+              </div>
+              <div>
+                <img
+                  src={MicroarrayIcon}
+                  className="experiment__stats-icon"
+                  alt="MicroArray Badge Icon"
+                />{' '}
+                {experiment.id}
+              </div>
+            </div>
+
+            <h4 className="experiment__title">
+              Submitter Supplied Information
+            </h4>
+
+            <div>
+              <div className="experiment__row">
+                <div className="experiment__row-label">Description</div>
+                <div>{experiment.description}</div>
+              </div>
+              <div className="experiment__row">
+                <div className="experiment__row-label">PubMed ID</div>
+                <div>{experiment.pubmed_id}</div>
+              </div>
+              <div className="experiment__row">
+                <div className="experiment__row-label">Publication Title</div>
+                <div>{experiment.publication_title}</div>
+              </div>
+              <div className="experiment__row">
+                <div className="experiment__row-label">
+                  Submitter’s Institution
+                </div>
+                <div>{experiment.submitter_institution}</div>
+              </div>
+              <div className="experiment__row">
+                <div className="experiment__row-label">Submitter</div>
+                <div>{experiment.pubmed_id}</div>
+              </div>
+            </div>
+
+            <hr className="experiment__divider" />
```

Comment:
hmmm, might want to do this border in CSS: https://stackoverflow.com/questions/3513576/correct-usage-of-html5-hr-tag?noredirect=1&lq=1